portal-kernel/src/com/liferay/portal/kernel/util/SystemEnv.java

```diff
@@ -27,7 +27,11 @@ public static void setProperties(Properties properties) {
 		Map<String, String> env = System.getenv();
 
 		for (Map.Entry<String, String> entry : env.entrySet()) {
-			properties.setProperty("env." + entry.getKey(), entry.getValue());
+			String key = entry.getKey();
```

Comment:
@W2jx After a second thought I think you don't need to add this, because it does not filter out the env override properties for portal properties (`LIFERAY_*`).

Reply:
@W2jx Also, your commit message should be "Apply the new util to SystemProperties...", not "to SystemEnv".

Reply:
Ok, I'll fix them.